integration-tests/rita.py

```diff
@@ -341,7 +339,7 @@ def read_email(node):
             mail = json.load(mail_file_handle)
             if mail["envelope"]["forward_path"][0] == "{}@example.com".format(id):
                 return ''.join(chr(i) for i in mail["message"])
-    print("cannot find email for node {}".format(id))
+    raise Exception("cannot find email for node {}".format(id))
```

Comment:
I remember from other PR that this change introduces a side effect. Is it intentional?

Reply:
Failures should generally exit. I was previously using `sys.exit(1)` rightaway. Come to think of it an exception might be better for just running `teardown()` rightaway. I didn't pay that much attention because I have rita.py overhauled in a nother branch.